src/api/authApi.js

```diff
@@ -0,0 +1,57 @@
+import config from "../config/config";
+
+export const getAuthUser = async () => {
+  const request = {
```

Comment:
```suggestion
try {
  const request = {
```
트라이 를 최상단에 선언해서 하는게 코드가 더 깔끔하게 보일 것 같아요

Reply:
아래 논의한 대로 해당 함수를 사용하는 컴포넌트에서 에러 처리하도록 여기서는 `try...catch` 제거하겠습니다!